test/e2e/storage/csi_volumes.go

```diff
@@ -266,3 +253,17 @@ func (m *manifestDriver) finalPatchOptions(f *framework.Framework) utils.PatchCS
 	}
 	return o
 }
+
+func (m *manifestDriver) GetVolume(config *testsuites.PerTestConfig, volumeNumber int) (map[string]string, bool, bool) {
+	attributes := map[string]string{"size": m.claimSize, "nsmode": "fsdax"}
+	shared := false
+	readOnly := false
+
+	return attributes, shared, readOnly
+}
+
+func (m *manifestDriver) GetCSIDriverName(config *testsuites.PerTestConfig) string {
+	// Return real driver name. 
+	// We can't use m.driverInfo.Name as its not the real driver name
+	return "pmem-csi.intel.com"
```

Comment:
`manifestDriver` was meant to be a configurable implementation implementation of the test driver interface. Hard-coding "pmem-csi.intel.com" here is inconsistent with that.

Two options:
- keep the current approach an introduce a m.CSIDriverName field that gets set to "pmem-csi.intel.com"
- remove the manifestDriver struct and replace the implementation with something that is specific to PMEM-CSI

Reply:
Chose option 1. Done